src/main/java/com/cst438/controller/StudentController.java

```
@@ -15,21 +16,36 @@
 @RestController
 @CrossOrigin(origins = "http://localhost:3000")
 public class StudentController {
-
+    @Autowired
+    private EnrollmentRepository enrollmentRepository;
 
    // student gets transcript showing list of all enrollments
    // studentId will be temporary until Login security is implemented
    //example URL  /transcript?studentId=19803
    @GetMapping("/transcripts")
    public List<EnrollmentDTO> getTranscript(@RequestParam("studentId") int studentId) {
-
-       // TODO
-
-       // list course_id, sec_id, title, credit, grade in chronological order
-       // user must be a student
-	   // hint: use enrollment repository method findEnrollmentByStudentIdOrderByTermId
-       // remove the following line when done
-       return null;
+       List<Enrollment> enrollments = enrollmentRepository.findEnrollmentsByStudentIdOrderByTermId(studentId);
```

Need to throw an error when studentId not found.